src/Controls/src/Core/Compatibility/Handlers/TableView/Android/TableViewRenderer.cs

```diff
@@ -107,7 +137,10 @@ public override SizeRequest GetDesiredSize(double widthConstraint, double height
 							continue;
 						}
 
-						AView listItem = _adapter.GetView(i, null, Control);
+						var platformView = cell.Handler?.PlatformView as AView;
+						var convertView = (platformView?.Parent as AView) ?? platformView;
```

Comment:
Should use here an extension method searching a parent of an specific type?

Reply:
I don't think we need to complicate it that much. The cells are only ever added to `ConditionalFocusLayout`. That's the only type that we use inside the TableView